### PR TITLE
Add graph-backup-job component

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/thoth-application/issues/58

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: graph-backup-job/base/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                - name: THOTH_DEPLOYMENT_NAME
+                  valueFrom:
+                    configMapKeyRef:
+                      key: storage-bucket-name
+                      name: thoth
+                - name: THOTH_S3_ENDPOINT_URL
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-host
+                      name: thoth
```
Comment (pacospace) [inline graph-backup-job/base/cronjob.yaml:36]: `ceph` has a specific CM in core

File: graph-backup-job/base/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    name: thoth
+                - name: SENTRY_DSN
+                  valueFrom:
+                    secretKeyRef:
+                      name: thoth
+                      key: sentry-dsn
+                - name: PROMETHEUS_PUSHGATEWAY_URL
+                  valueFrom:
+                    configMapKeyRef:
+                      name: thoth
```
Comment (pacospace) [inline graph-backup-job/base/cronjob.yaml:75]: also `prometheus` has a specific CM

File: graph-backup-job/base/cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                      key: metrics-pushgateway-url
+                - name: KNOWLEDGE_GRAPH_HOST
+                  valueFrom:
+                    configMapKeyRef:
+                      key: postgresql-host
+                      name: thoth
```
Comment (pacospace) [inline graph-backup-job/base/cronjob.yaml:81]: this should be added to `postgresql` CM in core

**Comment by harshad16** [comment]
my bad, please have a look at above comments

**Comment by fridex** [comment]
Adjusted. Does it mean we keep templates consistent now? :)

**Pull request merged**